### DISCUSSION TITLE
Upgrade python version for peering-manager docker hub image

### DISCUSSION THREAD

**Discussion by MaxBear** [peering-manager/peering-manager#843] [Q&A] [answered]
I noticed the version of python used for peering-manager docker hub images still use python3.9 , eg. lates release  `peeringmanager/peering-manager:latest-ldap` This is quite out of date. Any plans to package peering-manager with python 12 or 11 ?

**Comment by gmazoyer** [comment, marked as answer]
Peering Manager 1.8 does not support Python 3.12, this will have to wait for Peering Manager 1.9 which is the next release planned. The Docker image will be updated accordingly.

**Reply by MaxBear** [reply]
Thanks for your answer, when do you plan to release pm 1.9 ?

**Reply by gmazoyer** [reply]
No ETA, when it'll be ready :)

**Reply by MaxBear** [reply]
Thanks !
